common/metrics/build.rs

```diff
@@ -1,14 +1,20 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+use std::env;
 use std::process::Command;
 
 /// Save revision info to environment variable
 fn main() {
-    let output = Command::new("git")
-        .args(&["rev-parse", "--short", "HEAD"])
-        .output()
-        .unwrap();
-    let git_rev = String::from_utf8(output.stdout).unwrap();
-    println!("cargo:rustc-env=GIT_REVISION={}", git_rev);
+    match env::var("GIT_REVISION") {
```

Comment:
I think we should be able to just do an `if env::var("...").is_err()` and not set anything if its already set, instead of resetting it. might have to test that though